static/app/views/dashboardsV2/data.tsx

```diff
@@ -2,8 +2,10 @@ import {t} from 'app/locale';
 
 import {DashboardDetails} from './types';
 
+export const emptyDashboardId = 'default-overview';
```

Comment:
This dashboard id (`default-overview`) refers to one of the hardcoded pre-built dashboards that comes with widgets. So it's not actually empty.

Reply:
I see...but I noticed that it's not possible to delete the pre-built dashboard too, so this is always the "default"

Reply:
@priscilawebdev We allow users to delete the pre-built dashboard by using "tombstones"; by marking it as deleted for an org.